interfaces/acados_template/acados_template/acados_ocp_options.py

```diff
@@ -735,6 +737,28 @@ def reg_epsilon(self):
         """Epsilon for regularization, used if regularize_method in ['PROJECT', 'MIRROR', 'CONVEXIFY']"""
         return self.__reg_epsilon
 
+    @property
+    def reg_max_cond_block(self):
+        """Maximum condition number per block when using regularize_method in ['PROJECT', 'MIRROR'] with reg_adaptive_eps = True
```

Comment:
Could be even more explizit: 
Maximum condition number of each Hessian block after regularization with regularize_method in ['PROJECT', 'MIRROR'] and with reg_adaptive_eps = True